mmv1/products/alloydb/api.yaml

```diff
@@ -0,0 +1,412 @@
+
+              A duration in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s".
+          - !ruby/object:Api::Type::String
+            name: "location"
+            description: |
+              The location where the backup will be stored. Currently, the only supported option is to store the backup in the same region as the cluster.
+          - !ruby/object:Api::Type::KeyValuePairs
+            name: "labels"
+            description: "Labels to apply to backups created using this configuration."
+          - !ruby/object:Api::Type::NestedObject
+            name: "weeklySchedule"
+            input: true
+            description: "Weekly schedule for the Backup."
+            properties:
+              - !ruby/object:Api::Type::Array
+                name: "daysOfWeek"
+                description: |
+                  The days of the week to perform a backup. At least one day of the week must be provided.
+                item_type: !ruby/object:Api::Type::Enum
+                  name: "daysOfWeek"
+                  description: |
+                    The days of the week to perform a backup. At least one day of the week must be provided.
+                  values:
+                    - :DAY_OF_WEEK_UNSPECIFIED
+                    - :MONDAY
+                    - :TUESDAY
+                    - :WEDNESDAY
+                    - :THURSDAY
+                    - :FRIDAY
+                    - :SATURDAY
+                    - :SUNDAY
+              - !ruby/object:Api::Type::Array
+                name: "startTimes"
+                description: |
+                  The times during the day to start a backup. At least one start time must be provided. The start times are assumed to be in UTC and to be an exact hour (e.g., 04:00:00).
+                item_type: !ruby/object:Api::Type::NestedObject
+                  properties:
+                    - !ruby/object:Api::Type::Integer
+                      name: hours
+                      description: |
+                        Hours of day in 24 hour format. Should be from 0 to 23. An API may choose to allow the value "24:00:00" for scenarios like business closing time.
+                    - !ruby/object:Api::Type::Integer
+                      name: minutes
+                      description: |
+                        Minutes of hour of day. Must be from 0 to 59.
+                    - !ruby/object:Api::Type::Integer
+                      name: seconds
+                      description: |
+                        Seconds of minutes of the time. Must normally be from 0 to 59. An API may allow the value 60 if it allows leap-seconds.
+                    - !ruby/object:Api::Type::Integer
+                      name: nanos
+                      description: |
+                        Fractions of seconds in nanoseconds. Must be from 0 to 999,999,999.
+          - !ruby/object:Api::Type::NestedObject
+            name: "timeBasedRetention"
```

Comment:
Regarding `conflicts` - it looks like that's not currently supported for fields nested inside other fields. I've opened https://github.com/hashicorp/terraform-provider-google/issues/12052 to track this issue. For this PR, leaving it off is fine.

Nothing to change here, just wanted to let you know what I found.

Reply:
The issue with `conflicts` was resolved in https://github.com/GoogleCloudPlatform/magic-modules/pull/6276 so it would probably be possible to add back in if you want to, but no worries either way, already a lot going on here.

Reply:
done